scripts/entrypoint.sh

```diff
@@ -91,7 +94,7 @@ fi
 
 log_in_json info "Copying CNI plugin binary and config file ... "
 
-install aws-cni "$HOST_CNI_BIN_PATH"
+#install aws-cni "$HOST_CNI_BIN_PATH"
```

Comment:
I think you should remove this line. If you still have reasons to keep it, add some comment so that reader has context.

Reply:
Removed it :)